rustainers/src/runner/runner_network.rs

```diff
@@ -0,0 +1,60 @@
+use serde::{Deserialize, Serialize};
+
+/// The network driver
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum Driver {
+    /// Bridge
+    Bridge,
+    /// Host
+    Host,
+    /// Overlay
+    Overlay,
+    /// Ipvlan
+    Ipvlan,
+    /// Macvlan
+    Macvlan,
+    /// Null
+    Null,
+}
+
+/// A runner network
+///
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct RunnerNetwork {
```

Comment:
🔨 **todo**: This type could be moved into the `crate::container::network` module.
Also move the related tests to remove this file